core/domain/exp_domain.py

```diff
@@ -454,13 +454,17 @@ class ExplorationVersionsDiff(python_utils.OBJECT):
 
     Attributes:
         added_state_names: list(str). Name of the states added to the
-            exploration from prev_exp_version to current_exp_version.
+            exploration from prev_exp_version to current_exp_version. It stores
+            the newest name of the added state.
```

Comment:
Maybe rephrase it to 'Stores newest names of added states'? Currently, it seems like it is only storing a single str value.

Reply:
Done.